lambda-runtime/README.md

```diff
@@ -0,0 +1,14 @@
+# lambda-runtime
+
+An opinionated runtime environment for haskell applications running on [AWS Lambda].
+
+Key Features:
+
+- Only exposes a single function that constructs a lambda
+- Strong emphasis on error catching and handling, exposing the ability to catch `InternalLambdaClientErrors` to user applications for post processing and reporting.
```

Comment:
@epicallan I would suggest to either `s/to user applications/in user applications/` or `s/.../from user applications/` (I'm no native english speaker though)

Reply:
@snusnu I think "in user applications" is probably better English.

Reply:
I agree with you on this. Going to make this update.